packages/javascript/bh-shared-ui/src/constants.ts

```diff
@@ -14,9 +14,248 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+import { Theme } from '@mui/material';
+import { addOpacityToHex } from './utils';
+
 export const NODE_GRAPH_RENDER_LIMIT = 1000;
 
 export const ZERO_VALUE_API_DATE = '0001-01-01T00:00:00Z';
 
 export const TIER_ZERO_TAG = 'admin_tier_0';
 export const TIER_ZERO_LABEL = 'High Value';
+
+export const lightPalette = {
+    primary: {
+        main: '#33318F',
+        dark: '#261F7A',
+    },
+    secondary: {
+        main: '#1A30FF',
+        dark: '#0524F0',
+    },
+    tertiary: {
+        main: '#5CC791',
+        dark: '#02C577',
+    },
+    color: {
+        primary: '#1D1B20',
+        links: '#1A30FF',
+        error: '#B44641',
+    },
+    neutral: {
+        primary: '#FFFFFF',
+        secondary: '#F4F4F4',
+        tertiary: '#E3E7EA',
+        quaternary: '#DADEE1',
+        quinary: '#CACFD3',
+    },
+    background: {
+        paper: '#fafafa',
+        default: '#e4e9eb',
+    },
+    low: 'rgb(255, 195, 15)',
+    moderate: 'rgb(255, 97, 66)',
+    high: 'rgb(205, 0, 117)',
+    critical: 'rgb(76, 29, 143)',
```

Comment:
nit: As a dev I don't find it immediately obvious why I would use `primary.main` over `color.primary` over `neutral.primary`

Similarly if this is the `lightPalette` why are there options like `primary.dark`

TLDR I think we can discuss some better naming conventions here

Reply:
This is a result of trying to fit the way we defined our styles in the design system with how MUI has their styles defined while doing this for the hackathon. I agree things can and should be clearer but hopefully we move from MUI soon enough that this isn't in some weird in between state.

Reply:
I'll add it to the list!